core/src/main/java/org/apache/calcite/rel/rel2sql/RelToSqlConverter.java

```diff
@@ -1114,6 +1116,38 @@ private void parseCorrelTable(RelNode relNode, Result x) {
     }
   }
 
+  /** Visits a RavenDistinctProject; called by {@link #dispatch} via reflection. */
+  public Result visit(RavenDistinctProject e) {
```

Comment:
We had agreed earlier to not add any more code related to RavenDistinctProject until the model issues have been resolved. @abhisheikpushp @kk19ag 
Can you please look at this